lib/generate-id.js

```diff
@@ -1,17 +1,16 @@
-const { generateId } = require( './utils' )
-const colors = require( './string-colors' )
+const commander = require( 'commander' )
 
+const { generateId } = require( './utils' )
 const { Shabads, Lines } = require( '../' )
 
+const availableTypes = [ 'shabad', 'line' ]
```

Comment:
This is actually a duplicate of `idTypes`, since `Object.keys(idTypes)` will give you the exact same, and you don't want to be maintaining two lists if you can help it. So just use the aforementioned where you need it, since I can see 1 place, and remove this var